ball_tracker_rgb:
  ros__parameters:
    image_topic: "/camera/image_raw"
    camera_info_topic: "/camera/camera_info"
    # Tennis ball HSV (tune later to your lighting/ball)
    hsv_lower: [25, 140, 140]
    hsv_upper: [45, 255, 255]
    min_area_px: 80
    det_ball_topic: "/detections/ball"
    debug_image_topic: "/camera/image_debug"
    mask_image_topic: "/camera/image_mask"

people_detector:
  ros__parameters:
    image_topic: "/camera/image_raw"
    camera_info_topic: "/camera/camera_info"
    det_topic: "/detections/person"
    model_path: "/ws/ros2_ws/models/yolov8n.onnx"  # <â€” inside the container
    input_size: [640, 640]
    conf_threshold: 0.25
    iou_threshold: 0.45
    infer_every_n: 2
    person_class_ids: [0]
